apps/content_types/content_types.py

```diff
@@ -111,6 +109,9 @@ def on_create(self, docs):
         for doc in docs:
             self._set_updated_by(doc)
             self._set_created_by(doc)
+            if 'schema' not in doc and 'editor' not in doc:
+                doc['schema'] = DEFAULT_SCHEMA
+                doc['editor'] = DEFAULT_EDITOR
```

Comment:
if there is "schema" and no "editor" or vice/versa there will be no default value, is it expected ?